### DISCUSSION TITLE
Power Automate Bypass

### DISCUSSION THREAD

**Discussion by ghost** [Innofactor/XrmToolBox.BulkDataUpdater#247] [Q&A]
Hello! I was wondering if there is any way to use the "SuppressCallbackRegistrationExpanderJob" parameter for Power Automate flows in Bulk Data Updater. I haven't used it for a while and the UI directed me to the learn documentation for sync and async. I don't see a toggle in the app for it, but wanted to make sure.

Thank you! This tool is incredible!
Best - AG

**Comment by rappen** [comment]
I'm extremely sorry about this very very late response, it slipped between the cracks somewhere.

I added the parameters about sync/async that you can disable, but I've not heard about `SuppressCallbackRegistrationExpanderJob`... I see I need to go read about that feature!
https://learn.microsoft.com/power-apps/developer/data-platform/bypass-power-automate-flows

**Comment by Bob-Capel** [comment]
Hey, Jonas! Have you had a chance to consider this? 
BulkDataUpdater is an awesome tool. This feature would help a lot.
Thanks!
